runtime/gc_structs/ConstantPoolClassSlotIterator.cpp

```diff
@@ -60,9 +60,10 @@ GC_ConstantPoolClassSlotIterator::nextSlot()
 		_cpDescriptionIndex -= 1;
 
 		/* Determine if the slot should be processed */
-		if(slotType == J9CPTYPE_CLASS) {
-			return &(((J9RAMClassRef *) slotPtr)->value);
+		if (slotType == J9CPTYPE_CLASS) {
+			classPtr = ((J9RAMClassRef *) slotPtr)->value;
+			if (NULL != classPtr) break;
```

Comment:
According Coding Standards { } brackets must be used even in case of single operator inside logical if